lua/entities/ent_jack_gmod_humans.lua

```diff
@@ -93,14 +93,14 @@ if(SERVER)then
 				npc:SetKeyValue("citizentype","3")
 				npc:SetKeyValue("Expression Type","Random")
 				npc:SetKeyValue("additionalequipment","wep_jack_gmod_npcrocketlauncher")
-				npc:SetCurrentWeaponProficiency(WEAPON_PROFICIENCY_PERFECT) --but gosh damn do they suck giant horse schlong at shooting things with their RPG
-				npc:SetKeyValue("spawnflags","524544") --256+524288(SF_CITIZEN_AMMORESUPPLIER)
+				npc:SetCurrentWeaponProficiency(WEAPON_PROFICIENCY_PERFECT)
+				npc:SetKeyValue("spawnflags","524544") --256+524288(SF_CITIZEN_AMMORESUPPLIER)
 				npc.JackyFinickyAmmoGiver=true
 				npc:SetKeyValue("ammosupply","RPG_Round")
 				npc:SetKeyValue("ammoamount","1")
 				npc:Spawn()
 				npc:Activate()
-				npc:SetKeyValue("SquadName",SquadName)
+				npc:SetKeyValue("SquadName",SquadName)

```

Comment:
these lines are exactly the same what is goin on

Reply:
Yeah I'm wondering the same